### PR TITLE
Added `SQLServerToParquet` flow.

### PR DESCRIPTION
**Pull request by angelika233**
## Summary
Added `SQLServerToParquet` flow.

## Importance
New flow for transferring data from SQLServer database to Parquet file

## Checklist

This PR:
- [ ] follows the guidelines laid out in `CONTRIBUTING.md`
- [ ] links relevant issue(s)
- [x] adds/updates tests (if appropriate)
- [x] adds/updates docstrings (if appropriate)
- [x] adds an entry in `CHANGELOG.md`

### CODE REVIEW

**Review by adrian-wojcik — APPROVED** [review]
Everything looks good.

File: viadot/flows/sql_server_to_parquet.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        sql_query: str,
+        local_file_path: str,
+        sqlserver_config_key: str = None,
+        if_exists: Literal["fail", "replace", "append", "skip", "delete"] = "fail",
+        timeout: int = 3600,
+        *args: List[any],
+        **kwargs: Dict[str, Any],
+    ):
+        """
+        Flow for upolading data from SQL Server to Parquet file.
```
Comment (Rafalz13) [inline viadot/flows/sql_server_to_parquet.py:22]: typo -> `uploading`

File: viadot/flows/sql_server_to_parquet.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+        Args:
+            name (str): The name of the flow.
+            sql_query (str, required): The query to execute on the SQL Server database. If don't start with "SELECT"
```
Comment (Rafalz13) [inline viadot/flows/sql_server_to_parquet.py:26]: Is writing `required` mandatory? 
For `name` and `local_file_path` you have only type.

File: tests/integration/flows/test_sql_server_to_parquet.py
Comment (Rafalz13) [inline tests/integration/flows/test_sql_server_to_parquet.py:1]: Please add a test to check the functionality of this flow. Check if the parquet file is created.

**Review by Rafalz13 — COMMENTED** [review]
I added comments.

File: viadot/flows/sql_server_to_parquet.py (hunk 2 above)
Comment (angelika233) [reply viadot/flows/sql_server_to_parquet.py:26]: I added `required` for `name` and `local_file_path`

**Review by Rafalz13 — APPROVED** [review]

**Pull request merged**